src/main/steps/applicant/upload-document/upload-your-documents/content.test.ts

```diff
@@ -7,19 +7,69 @@ import { generateContent } from './content';
 const en = {
```

Comment:
why cant the parent language contents be imported and used?

Reply:
Changed to import the language contents as suggested